.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3-jessie-node-browsers
+    steps:
+      - checkout
+      - run: bundle install
```

Comment:
Premature optimization: 

- we can use cache here:

```
    - restore_cache:
        keys:
        - v2-dependencies-{{ checksum "Gemfile.lock" }}
```

- and after we `bundle exec...`:
```
    - save_cache:
        paths:
        - vendor/bundle
        key: v2-dependencies-{{ checksum "Gemfile.lock" }}
```

> **Note**: I am not ruby person and do not know internals of `bundle`, I quickly googled for the default installation path, it may be wrong.

Reply:
You are correct, but you also need to specify `--deployment` to `bundle install` because by default on a development machine it will install the packages system-wide afaik.